### PR TITLE
Economy 2019 Part C Branch A

### PR DESCRIPTION
**Pull request by Kurfursten**
The main economy reform was too vulnerable to blocking because it lacked atomicity. I'm going to split it in three to make it easier to push through. Part A is aimed at accounts issues, Part B will target vending machine improvements, and Part C will be where we take on the miner metaclub.

Miners asked me to instead just lower the values of minerals instead of going to a point system. This will disrupt mining vend progression, but I'm willing to put it up to see if they vote for it. I'll also make a branch B with the points. Personally, I like the points and I don't care what we call them. They are smoother for everyone. No disruption of your vendor progression and no worries about further adjusting prices down the line.

* Iron/Sand/Plasma: 0.2 credits each (mining one tile gets you 1 credit)
* Uranium/Silver/Gold/Bananium/Phazon: 1 credits each (5 credits per tile)
* Diamond: 4 credits (20 per tile)

Vendor

* Canary 100 -> 20
* Chili 100 -> 20
* Cigar 100 -> 20
* Knife 150 -> 15
* Whiskey 150 -> 15
* Soap 150 -> 15
* Facehugger 250 -> 25
* Laz Belt 500 -> 50
* Pokeball 250 -> 50
* Laz 1000 -> 50 (discuss? may make breaking into mining to revive pets too common)
* Jackhammer 500 -> 50
* Drone 500 -> 50
* Jaunter 250 -> 25
* Resonator 750 -> 75 (discuss? this might be too cheap considering its damage but nobody seems to use them)
* Money Bag 1000 -> 10
* PKA 1000 -> 300
* Jetpack 2000 -> 300
* Hookshot 3000 -> 300
* Adv Laz 3000 -> 150
* Silver Pick 1000 -> 100
* Gold Pick 2000 -> 200
* Diamond Pick 3000 -> 300
* Borg Hookshot 3000 -> 300

🆑 
* tweak: The values of minerals at the smelter have been dramatically reduced. Mining vendor prices have all dropped.

### CODE REVIEW

**Comment by ghost** [comment]
Lets say a miner mines 50 of each resource except phazon and diamond, and then somehow manages to find 10 diamond deposits.

Our miner has just brought in 250 uranium sheets, 250 gold bars, 250 silver bars, 250 glass sheets, 250 iron sheets, 250 plasma sheets and 50 diamonds. Miner is rewarded with 387.5 credits to buy 1 jetpack after an hour of work. Meanwhile science can now equip every single crewmember with ablative armor and an assault rifle with 2 mags, or build an army of mechs. Assuming the round doesn't end in the next few minutes after our miner buys their jetpack.

Here's chances for various ore spawns taken directly from mine_turfs.dm based on their weighted list distributions.

Regular deposit
- 46% iron
- 23% plasma
- 4% uranium/gold/silver
- 1% diamond

High Chance deposit
- 34% iron
- 28% plasma
- 11% uranium/gold/silver
- 2% diamond

Credit rewards should be derived directly from the above percentages to represent effort spent. However I think this PR is a better approach than introducing a third currency type.

**Comment by DrSnips** [comment]
I personally think the good boy points system would be better since it allows miners to still have big values to make them feel special without having to worry about the future of the economy ever again.

**Comment by Kurfursten** [comment]
If you support points you can vote for it here: #22307 - Unfortunately it does not look like we're going that direction.

@codeungabunga I don't think your numbers are factoring in vein size. As any regular miner knows, iron sand and plasma have very large veins while the others do not. We also know that a diamond drill makes 50 tiles more of a 1 minute job than a 1 hour job. Yes, the drill does mine near instantly but I'm giving our hypothetical miner an entire 1.2 seconds to sluggishly pick up the goods with his bag (even though it's one click where his mouse should be already hovering).

**Comment by ghost** [comment]
Did you run statistical analysis on the amount of ores available on average on box asteroid? I have at least put in a modicum of effort towards numerically estimating ore distribution, your numbers are completely arbitrary.

**Comment by Tokanova** [comment]
This is fine
t. miner metaclubber

**Comment by SGuivenne** [comment]
Will plasma still be sellable by cargo for additional money?  I like the idea of this if it can be used to encourage QM to actually send plasma to centcomm for additional funds for miners and themselves.

**Comment by Kurfursten** [comment]
I haven't changed anything regarding the cargo plasma

**Comment by Kurfursten** [comment]
re: ungabunga

I was not able to gather statistics on iron or plasma because there were too many spawns for an sqdl query. I was able to query diamond deposits but to prevent lag sqdl seems to have failsafes against queries that would return a very large number of results. I also considered hand counting the ores but that seemed stupid very quickly after I took some screenshots.

This is a completely random average round on Box. It should look like a familiar sight to miners. These locations aren't chosen particularly rigorously I just took a few pics to show there are hundreds of iron and plasma ores.
normal mining shift

**Comment by ghost** [comment]
I'm not denying that there are hundreds of ore deposits, notice that the percentages I wrote represent what's shown on your pictures. My point is that having gold/uranium/silver priced at 125% cost of iron/plasma is bad considering the quantities involved. How can 2 tiles of sand be worth more than 1 tile of rare ores? 5 sand tiles worth as much as one diamond deposit?

Gold/uranium/silver should give significantly more reward for miners, diamond even more so. Phazon price is also very low, why is it equal to 2 sand tiles when it's practically the rarest ore to find? Don't forget about telecrystal ores, bananium and mythril.

You have set telecrystals and mythril to give no credits at all.

**Comment by aceedex** [comment]
I like it like this. A little weird on the prices though. Like you mentioned in the other PR PKAs are a bit of an issue, their price is rough. Any thoughs on moving the mininvends to  the roid like I mentioned? Tbh doing this would make pricing a lot easier since you dont have to account for break ins at all

Another thing that I find a little odd is that plasma was dunked to the lesser payouts, understandably so, but I believe that that money could be partially out towards diamonds.

**Comment by Kurfursten** [comment]
Sure, U/G/S could be tweaked up a bit, perhaps 2 credits a tile like bananium. Also with the results of my SQDL query I'm satisfied diamond could be worth more than it is (10 or 20 credits a tile, maybe). Naturally telecrystal and mythril shouldn't receive any incentivization by your own logic since they aren't useful to the station.

I don't think moving it to the roid solves the money problem *entirely* since its newly cheapened price will attract scientists and traders to smash in if the price was exceptionally low (like, 15 credits or something... I think it's fine at its current price) - although I will say that putting the mining vends on the main station was always a conscious choice to encourage more trips back to the station so miners spend less time isolated and return more often - to interact with others and to expose them for assassination objectives. I don't think I would oppose a mapping PR to move them but I would definitely have mixed feelings about it. I still think the labor points system is vastly superior to this.

Especially given that people want to still be able to hack machines to get free things, this option is just worse in every way than labor points. Maybe a better compromise would be to let you buy credits with labor points at some exchange rate. The minerclub problem is a tough one to break. I think one way to do it might be to offer to let you buy credits for some high amount of labor points, but I think the silent crew have firmly set themselves against that. I really do think it's the best solution for keeping the experience intact though.

**Comment by ghost** [comment]
I didn't base my logic on usefulness but rarity, as I have repeatedly stated. Telecrystals are very useful to science and telescience for subspace tunneler and restricted area teleports respectively.

Labor points are not part of this PR.

**Comment by agentofzog2** [comment]
i feel like c*ders should be forced to spend several rounds mining before they make it so you have to mine a boatload of ore in order to be allowed to kill that goliath that wandered by the outpost (then wonder why not just order eguns, which you can then upgrade into AEGs with all the excess materials). 

See the thing about mining is that it's boring as shit, I can stripmine the whole roid if I want to but I'd rather just get the bare minimum science needs to get the advanced stuff and then fuck off and do something else if there's something happening on station.

 I'm having a hard time remembering but a couple full stacks of iron/glass/plasma and a half stack each of gold/silver/uranium plus 5-15 diamonds  currently gets you around a thousand credits which is enough for a PKA if you wanna be good boi and as long as the roid remains boring I'd prefer to not have to spend several times as much time to get the basic tools I need to maybe have a modicum of fun there.

**Comment by ghost** [comment]
I would very much like to see the results of @Kurfursten playing a miner round and getting a PKA (a extremely basic piece of mining gear) under these new prices.

**Comment by DrSnips** [comment]
As far as I'm aware plasma4cash is unchanged and nets you a decent chunk of money. I guess it's time for you to take advantage of it.

**Comment by Kurfursten** [comment]
I offered you labor points which do not disrupt your PKA purchasing or mining progression at all. Labor points are a better system. I know this because I've played a lot of shaft miner in spite of what you think.

The informed player who understands that a miner feels right having 4 PKAs on his belt *soon* into the shift would naturally support branch B. I invite the highly vocal minority of ungabunga, zog, and tokanova to join me in support for labor points. It's not too late.

**Comment by bathosbathtime** [comment]
It is entirely possible to have miners paid in money and not have their current progression disrupted.  Please tweak the values of the payout to better reflect the work done instead of trying to push something that maybe two people want.

File: code/modules/mining/materials.dm [hunk 1]
```diff
@@ -205,7 +205,7 @@ var/global/list/initial_materials	//Stores all the matids = 0 in helping New
 /datum/material/plasma
 	name="Plasma"
 	id=MAT_PLASMA
-	value=40
+	value=0.2
```
Comment (bathosbathtime) [inline code/modules/mining/materials.dm:208]: Why is this and diamond's reduction so extreme compared to the other ores.  This is 200 times less valuable compared to iron and sand's time 5 reduction or gold, silver, and uranium's 80 times reduction.

File: code/modules/mining/materials.dm [hunk 2]
```diff
@@ -397,7 +397,7 @@ var/global/list/initial_materials	//Stores all the matids = 0 in helping New
 /datum/material/telecrystal
 	name="telecrystal"
 	id=MAT_TELECRYSTAL
-	value=200
+	value=0
```
Comment (bathosbathtime) [inline code/modules/mining/materials.dm]: Why was this set to zero, it can be mined.

**Comment by ghost** [comment]
>vocal minority

If you're not going to take this serious then I have nothing more to add.

**Review by bathosbathtime — COMMENTED** [review]
It is possible to have the shaft miners' money output toned down without absolutely kneecapping them and I believe you are more than smart enough to realize that.

**Comment by DrSnips** [comment]
Id be for test merging this to the test serb and having all of these people able to try it just so they can see what kind of numbers they pull

File: code/modules/mining/materials.dm (hunk 1 above)
Comment (Kurfursten) [reply code/modules/mining/materials.dm:208]: Plasma is the most abundant and least useful thing a miner can bring in. While I do see reason in bumping up G/S/U in value, I feel very good where the price of plasma has landed.

**Comment by Kurfursten** [comment]
I had a thought that might please miners. We could just move PKAs out of the vendor entirely and put them in mining lockers.

That would keep progression from being disrupted and avoid the "cheap guns in a vendor" problem. Lockers can be stolen+emittered/IED'd, yes, but I think that's a much greater barrier to entry than hacking a vendor or, if the price was lower, buying 15 credit guns or something similar.

**Comment by Hoshi-chan** [comment]
Kurf, you're making this worse and worse as this discussion goes on. It takes one full PKA, not missing a single shot, to kill a single goliath. Usually you carry two, or sometimes three. The payout for minerals is absurdly low under this PR. If you push this through, there will be _nobody_ playing mining in a month, I guarantee it.

I'm in full agreement with @agentofzog2 here. You should have to play mining for a week before you even consider pushing this through. Your attitude of "But I gave you a better option!" isn't helping things. You're not really giving us options; you're giving us barely any credits for a _fuckload_ of work, far more than any miner ever or will do, and now you want to remove PKAs from the mining vend. At this point there's no way the emojiocracy is accurate. I think if most miners read this thread they'd change their opinion.

**Comment by DrSnips** [comment]
Why the hell do miners get credits anyway besides getting stuff out of a vendor for a sense of progression? They get a wage just like everybody else.
Here's an idea. We remove the vendor completely and put it to sci or whatever. This way the amount of work and what they mine matters. You may not like this solution but you guys won't quit bitching about the amount of credits you get and the amount of credits that you want will absolutely destroy an economy after you get a diamond mining tool. You could lock the pka to mining access lock boxes in the proto. Skinning knives could just be moved to locker. Soap could come via cargo if it isn't already in the janitor crate.

**Comment by Kurfursten** [comment]
@Hoshi-chan I've played years of miner, which is why I support the labor points system. It's not too late to join me. I understand that you need those PKAs early and ideally to carry multiple of them. I've killed Angie, I know what a damage sponge she is. As Snips points out, associating guns with a vending machine was a mistake.

I think I have also found another option that might please everyone: instead of having a new currency, we could have the Smelter work like the Biogenerator. All the miners contribute to a shared smelter points value and you can get your PKAs and toys directly there. No more trips back to the station, ID locked, and a progression system just like other departments. Take out the credit middleman entirely. This would be a new screen on the smelter, and we would not lower the value of minerals at all because it would no longer be a source of inflation. Therefore, mining would not lose its sense of fast progression. I think this is perhaps even ***better*** than labor points overall and brings the department in line with other departments in the idea of a shared, communal progression benefit (biogenerator in botany, matter synth in engineering, shared tech levels in R&D, shared cargo account to contribute to as techies, etc.)

**Comment by SGuivenne** [comment]
The whole point of it being on station is to force the miners to make a return trip to station, wherein they may as well carry all that ore with them anyway rather than fuck off forever.   If you deprive miners of a reason to return to station they simply wont.

**Comment by DrSnips** [comment]
My solution would also bring them on station

**Comment by Kurfursten** [comment]
Could also put [pointbased printer] on the station where the vendors are and link it with radio, like how the smelter and smelter computer are radio linked

**Comment by SonixApache** [comment]
Slimes' view on the matter convinced me that labor points would be better than this solution.
Your idea of making the system in line with the other departments resonates with me even more. 
I'd go with that.

**Comment by Kurfursten** [comment]
Do you want me to make it a separate machine linked by radio or built straight into the smelter as a secondary screen?

**Comment by SonixApache** [comment]
I think making it a radio-linked machine would be better.
The whole "Makes miners return to the station" thing.

**Comment by ghost** [comment]
Miners currently return to station to get a diamond drill, ore loader and bag of holding from science. All of mining gear should be possible to print in science, having multiple ways of obtaining the same things is what causes varied gameplay. As opposed to locking things behind very specific interactions with no alternatives.

**Comment by Kurfursten** [comment]
I don't object to putting PKAs in the protolathe, it's not like science doesn't already have gun designs. It could even be a use for plasma since almost nothing uses it.

**Comment by SonixApache** [comment]
fuck the protolathe

**Comment by DrSnips** [comment]
I'm not going to claim I'm an advisor but if I were any part of mining I would advise you take option B if you really care about having things as similar to the past as possible because you sure as hell don't want to adapt to the new economy.

**Comment by Kurfursten** [comment]
There's no reason it can't go in both. I think code's suggestion of having multiple ways to get something holds weight in this case. There are some times I'd prefer for something to be exclusive somewhere (for example, I like having *some* things only available through cargo because that drives business for cargo), but I don't feel like the PKA falls in that class of items.

**Comment by DrSnips** [comment]
It seems I'm late to the party that you've already closed B. Oh well then.

**Comment by aceedex** [comment]
Aight hear me out
Ita stupid but its the best possible solution I can see
Go to minivend. Buy PKA voucher, go to room in the assteroid, put voucher in wall dispenser, receive PKA

**Comment by Kurfursten** [comment]
@DrSnips I don't think B was going to pass, but we'll go ahead with a C option that carries on the idea of moving PKAs to a protolathe/smelterprinter system. Not this weekend though, going to the lantern festival with gf.

@aceedex Actually I see what you're saying now, that could solve problems to some extent because it would gate PKAs behind having mining shuttle access, although on Deff that is not required to fly to the mining base. It still involves making a second voucher machine and I think people would complain because vouchers are "currency" like miningbux

**Comment by FudgePucker** [comment]
@Kurfursten 
"and I think people would complain because vouchers are "currency" like miningbux"
Stop being such a sore butt, that's the best solution to the PKA problem. Miners need like three KPAs to go pokemon hunting, you don't want people to have easy access to guns, so make them have to get a voucher for like 50 credits, then redeem it at the mining outpost. that's two separate areas you have to break into to get access to them, on different Z levels.

**Comment by DrSnips** [comment]
Personally I think department accounts were hit too hard but that's a separate issue from this PR

**Comment by FudgePucker** [comment]
Yea I just edited my comment.
The best way I think to sorta fix the economy would add in more ways to make cash PvE instead of only mining.
Inflation instead of making everybody poor.

**Comment by DrSnips** [comment]
The one issue is how clunky transactions [still] are. I recall registers, self vendors, and such getting a huge overhaul to improve their use but obviously that did nothing since they are hardly used if at all. I personally don't think you can do anything to fix how players use the economy. You can lead a horse to water but you cannot make it drink.

**Comment by SonixApache** [comment]
EFTPOS should be as simple as 
 - Swipe ID to link acocunt
 - Use in-hand to set price (and maybe description but i doubt anyone ever checks their bank records)
 - Swipe own ID to cancel
 - Swipe anyone else's ID to complete transaction

That'd solve most of those issues.

**Comment by Kurfursten** [comment]
@FudgePucker Calm down lad, you're new to the conversation so I don't expect you to know this, but Bathos was complaining about insertables like coins counting as a currency (in Branch B). He's the one you should be namecalling. Don't shoot the messenger.

**Comment by bathosbathtime** [comment]
Coinage is literally currency and miners can make a nigh-unlimitted amount with the coin Smith.
The voucher idea seems fine (good even) to me since it is extremely straight forward (buy thing to get thing somewhere else) instead of a new and convoluted system to be put on top of our already convoluted and largely ignored system of economy.

**Comment by Kurfursten** [comment]
I have nothing against vouchers but I agree they add an extra convoluted step. Normally its receive points spend points, with vouchers its receive cash spends cash go on a mystery journey to find out where your loot token is redeemable.

That's why I'd prefer to just redeem points straight at the smelter like the biogenerator. No floating snowflake vouchers, items, tokens, currencies. Just dump in your diamonds and receive guns.

**Comment by Hoshi-chan** [comment]
I want to contribute something constructive to this discussion, but I'm in a really bad headspace and all of my ideas are stupidly convoluted or involve adding new machines or mechanics. I can't think of any simple way to provide miners with their tools, give them some sort of monetary compensation for their work, and necessitate that they come to the station, since apparently that's important.

The important things here seem to be:
- Miners get their shit
- Science gets their shit
- Economy doesn't get fucked up by miners introducing an absurd amount of money into it

However, I think what's fucking me up, and maybe some others, is one more thing that is currently tied to mining, but is otherwise unrelated:
- There needs to be a way to manually earn money

If this economy update removes the way that mining is currently _the_ way to earn money (which it shouldn't, it should nerf it, but reasonably so), it also needs to add new ways to manually do so. Things like normal vending machine prices, the PDA upgrade console, the gift console outside cargo, the price of shit from the vox and the amount they're given for free, the payout of slot machines, and the price of goods at spessmart also need to be adjusted.

I think maybe some sort of mining requisition machine that tallies up an individual miner's contributions and opens up a stock of tools, for free, for that individual miner, would be cool. No points system, at least not one that's surfaced. I also think that miners should be compensated for their work, but the values proposed in the OP are a little too extreme. As @codeungabunga said, for about an hour of backbreaking work they're receiving maybe twice what other people are getting, passively, for doing nothing. I do get the desire to keep miners from inflating the economy and making money worthless though.

**Comment by bathosbathtime** [comment]
I said that the vouchers were not convoluted but okay.
The core of the issue (I have) is that I believe we should have more ways of manually earning money instead of less.  Mining shouldn't be the way of earning money, both in terms of being the best or being the only, and the other departments should be brought up instead of mining brought down(all the way).

**Comment by Hoshi-chan** [comment]
And my concern is that by removing the money-making potential of mining while adding nothing else, this winds up as one of those things that gets addressed "in a future PR" which never comes, so we wind up with no real way to earn money.

**Comment by DrSnips** [comment]
I don't really see the need for getting money from mining DIRECTLY. Miners, like everyone else, get a wage and by just existing they earn income hell they could even be dead and you just have their ID. We also have the plasma exchange thing with cargo still and it can turn out a good amount of money based on the new economy that won't destroy the economy from a usual round of mining. If it does come to the point where plasma returns destroy the world we can address it then.

**Comment by FudgePucker** [comment]
Engineers should be able to turn spare power in for money, even a conservative SME can produce over 40MW. A drizzle into the engineering department account for a few spare megawatts, out pacing the miners with diamond drill Ridleys with a super autistic engine array. I think goon had some type of mega laser shooting off into space to turn energy into cash. Would also be the coolest way to kill your target if traitor.

Botanists should be able to send new crop mutations back on the cargo shuttle for cash, the higher potency and yield means more cash for new mutation types, so building up stats before shipping them off would benefit more.

Medbay should just get paid more IMO, they are doctors not assistants.

Science should receive a bonus for reaching high research, upon discovering several new types of slimes, discovering large artifacts, and getting a high yield explosion in the bomb testing area. Bonuses instead of some feasible way to grind for money because their job is to do research, they should be paid for forward progress in different fields. Would also encourage them to do their jobs.

Janitors wages can fluctuate depending on how messy the station is, how many light bulbs are busted, how much trash is laying around, so on and so on.

Those are just a few ideas off of the top of my head. The other jobs can start charging people without it being a big deal if most of the station is loaded.

**Comment by bathosbathtime** [comment]
While I do agree we should be moving towards more manual ways to earn credits, I feel that we should focus more on mining right now to stay in scope.
I do feel that the prices for the ores need to be well thought out as it will be setting the pace for any future PRs that add ways oh earning money wether we want it to or not.
Timed paychecks (that also payout the same to everyone on payroll regardless of job or rank) is not an engaging form of earning money.

**Comment by Lunanite** [comment]
I see absolutely no reason why we need to change the entire system at all, just adjust the cash you get for smelting things to a reasonable amount, if you're so worried. Make it so that if a miner brings say, 15 minutes worth of mining through the smelter they can get a Jetpack and a PKA, with a bit of cash to spare? As you're proposing now, you'd have to bore yourself to death just to get a fucking PKA and not die to the damn Goliaths. No one would even bother stepping onto the Mining Shuttle.

**Comment by DrSnips** [comment]
On a side note I don't get why everyone is afraid of PKAs in the hands of a few other people. Currently they have a technical cost of 0, due to throwing, so it's not like everyone who wants one can't get one so how many people do you see running around on station with PKAs?

**Comment by JohnExplode** [comment]
Hey, so I fundamentally disagree with changing the system at all. I'm having trouble completely articulating my thoughts on this so please bear with me. I know the fear is are that the miners can put in an obscene amount of credits into the economy, however I do not believe the economy is exactly in a place where dumping a large amount of credits in actually matters. There simply does not seem to be a system where that much money actually makes a difference in the economy - science will still mass produce literally anything they need, a good mechanic will too. I do not see how it matters since mining is currently the ONLY way to make money on the station that is not through paychecks. There is hardly a buy and sell economy unless a Vox Trader is on the station and even then, the money is 100% arbitrary. My first point is that if you want to nerf mining - make the economy worth it first. The ONLY thing I ever spend money on as a Shaft Miner (which I've been playing nearly religiously before/after my multi-year break) is PKAs, a hookshot, food, a knife, and MAYBE jetpacks if I feel like I want to go looking for the clown roid. If this is the course we are going, then might I suggest just increasing the cost of items from the Mining Vending Machine instead?

IF you absolutely insist on changing the credits per unit or going to the voucher/point system, you need to make it so that we're not spending the entire round working towards our 1 jetpack or our 2 PKAs. 300 credits for 2 PKAs (I usually buy 4-5 depending if I get the Polar Star or not) which is something you absolutely need to survive out there. If this is going to be the case, then you better consider dropping the health values on the mobs on the roid or increasing the ammo count on the PKA, no miner wants to dump their entire magazine into a goliath and then run back and forth between the charging stations and back to where you previously were. If you are really going to do this, please make the asteroid actually fun and worth going to because as it stands (I posted about this recently) after you get through the first 35 minutes of mining there is almost nothing for a Miner to do on the roid that is "fun". If your job cannot endure the average round length (which I believe has gotten longer since I stopped playing in 2016, due to mult-antag) then it is frankly a boring job. If you have nothing to show for the amount of work you put into your job, then did you actually do anything? Did your shift even matter? Sure you got materials to every department and your 2 PKAs + a jetpack (wew!). But that's it. Meanwhile we got genetics creating gods, research pumping out plasma weapons and laser cannons, robotics creating Phazon mechs (both of these off of your work, most likely!), engineering has crazy autism projects they've now completed, mechanics have built literally whatever they want, etc etc. Even as is, mining is worth a decent amount of credits yet you have hardly anything to show for it. Only one person gets a Polar Star and if there are 3+ good miners, only one of them will probably get a fully reinforced hardsuit from goliath plating if only ~4 spawn. Honestly, if it takes 30 minutes just to get 2 PKAs (with all of your money) I'm just going to start asking science for 2 eguns or laser rifles instead.

Don't even try to say that miners get Ripleys because it frankly isn't even worth the use as is. Auto-ore loaders (which is my favorite addition to mining) has killed the use for those. And on a side note, need to be brought to the level of efficiency that Ore-loaders are.

tl;dr: make the economy actually worth using first, if you're gonna nerf anything maybe just increase vending machine costs. Also mining is braindead boring and the roid itself needs an overhaul. Everything is a bullet sponge (Angie was a cool and unique inclusion), but you basically have nothing to do after you get enough materials for Science + Mechanics + Robotics anyway. Also Ripleys suck just sayin

**Comment by ghost** [comment]
It's starting to feel like everything revolves around PKAs. Why are PKAs practical to use against players under normal station condition?

On other servers, PKA has one shot, self recharges and is really bad in atmosphere. Which translates to only being useful for fighting dumb NPCs in space which you can kite forever.

**Comment by JohnExplode** [comment]
> It's starting to feel like everything revolves around PKAs. Why are PKAs practical to use against players under normal station condition?
> 
> On other servers, PKA has one shot, self recharges and is really bad in atmosphere. Which translates to only being useful for fighting dumb NPCs in space which you can kite forever.

AFAIK they're not practical against player characters under normal conditions. In a vacuum they're very effective however - I think it's like 30 brute in those conditions. The problem isn't just the PKA nor does it revolve around it. The PKA is simply one of the most logical (and pretty much required) tools that you need. PKA should be used as a reference because it is likely the most bought item in the vending machine - everything else is honestly options compared to it. PKAs on other servers sound lame, and I do not believe they should be used as a reference point since things here are already done differently.

**Comment by aceedex** [comment]
Dont underestimate the PKA. Its 10 shots, each 15 in atmosphere 30 at under 50kpa or something. Thats a ranged toolbox!
Also slimes, throwing is simply not feasible. there's so much god damn junk in the miningvend to make throwing a guarantee

**Comment by JohnExplode** [comment]
> Dont underestimate the PKA. Its 10 shots, each 15 in atmosphere 30 at under 50kpa or something. Thats a ranged toolbox!

True! I don't mean to downplay its effectiveness in pressurized environments; its not the worst thing you can use in a fight. I also misread the suggest adjusted price for the PKA, my tired brain read it as "150" instead of the 300.

**Comment by Intigracy** [comment]
```
There simply does not seem to be a system where that much money actually makes a difference in the economy
```

Don't change the economy because miners will ruin it with their injection of cash
Don't change mining economy because there's no reason to since there's no economy
Don't change the economy because miners will ruin it with their injection of cash
Don't change mining economy because there's no reason to since there's no economy

**Comment by Hoshi-chan** [comment]
You're missing his point, Inti.

This is what is being proposed:
- Nerf the fuck out of mining's income, while adding nothing to compensate, and still nothing that makes money actually matter. Economy is "fixed" but still pointless. Mining is less fun.

This is what he thinks should happen:
- Add shit to make the economy matter before nerfing mining; things to purchase, other ways to make money, whatever. Once the amount of money matters, _then_ you nerf mining's income. Economy is fixed. Mining is less fun in some ways but hopefully more fun in others. Everyone gets to do capitalism gimmicks for a few weeks while the PR is fresh.

This PR is putting the cart before the horse.

**Comment by DrSnips** [comment]
I didn't know mining's whole "fun" came from the vendor. I guess the bag of holding, D.drill, and (lathe gun here) that they ask for nearly every round isn't the actual source of the fun.

**Comment by JohnExplode** [comment]
> I didn't know mining's whole "fun" came from the vendor. I guess the bag of holding, D.drill, and (lathe gun here) that they ask for nearly every round isn't the actual source of the fun.

Explain to me where the "fun" of mining comes from, in your opinion. 

To make it clear, the items do not make the job that much more fun, but it adds to the (little) fun you get to have. Mining for 45 minutes to get 1 gun or whatever is not fun. Not being able to fight against the monsters that are trying to murder you, is not fun. Even the drills don't make mining that much more fun. Items do well to marginally increase what you can do as a miner, and can make it just a bit more fun. Remember, after the first 15-20 minutes, most departments will have the amount of resources they need for the entire round, then most miners have nothing to do, except for play around with their mining toys.

**Comment by DrSnips** [comment]
Most miner end goals for "fun" are either:
* space adventure
* logging out in the roid dorms
* making the roid as flat as possible and dumping it in science
* maxing out their suit with hides via pokemon
* walking around the station in a hardsuit waiting for valids
Now lets see what they need for these.
* jetpack, bag of holding, gun. (EVA/vendor, sci, vendor/cargo order/sci/sec)
* nothing
* D.drill. (sci)
* Laz injectors, pokeballs. (vendor, vendor)
* gun, Ddrill. (cargo order/sci/sec, sci)
Of those activities only one of them solely relies on the vendor. So why do people want the vendor so much? Guns can be obtained for several places including the cargo console that they can approve themself and crack it open with gibtonite.

**Comment by JohnExplode** [comment]
The vendor provides the necessary tools to do your job safely and, as I said before, provide a marginal amount of fun. The fact is that mining is not a very dynamic or fun job. PKAs are necessary to remove threats, jetpacks are necessary to get to the clown roid. Hookshots are necessary such that I don't have to throw my tools every time I forget to toggle magboots and slip into space.

-Space Adventure
 These hardly last long enough to even justify. I guess you can go grab the gooncode, but that's hardly a challenge (or fun) if you've done it before/know where everything is. I'm not gonna ask to be let into EVA every round to get the jetpacks readily available for me in the vendor.

- Logging out in roid dorms
Never tried this one, might give it a whirl next time I get on.

- Making the roid as flat as possible
I've been doing this for years. It's boring, but knowing you have 350k by the end is the "fun" part if you can call it that.

- Maxing out suit w/ hides + pokemon
Correct me if I'm wrong, but I do not think that enough goliaths spawn to max everyone's suit out, especially when you have 2-3 miners every round.   Collecting monsters gives you something to do after you've killed the monster, but then what? If an antag comes to mess with you I guess you have better protection.

- valid hunting
Are miners space cowboys now? What happens when security doesn't want the local vigilante doing their job for them?

Also breaking crates open for guns in cargo is powergamey as fuck. You have access to allowed guns that are purchasable at the vendor. I use the PKA as my example because they're literally one of the tools you need to live out there. If you're bleeding out and a hivelord is nearby, a few PKA shots to kill that and eat the core are the difference between life and death.

If you're so keen on removing the vendor or whatever just throw the PKAs in the lockers, hardsuit storage or whatever. I don't want to powergame and beg science for guns or break open gun crates every round. I just want to use them to remove goliaths and rockernauts so I can safely get back to doing my job and not have to worry about getting tripped behind a stone wall.

Please look at what Hoshi said about my post. My primary concern is that if the economy/money changes happen to Shaft Miner they need something else to justify playing, and before that even happens they need to give the economy some weight and reason for the nerfs.

**Comment by DrSnips** [comment]
No one gives a fuck if you break open crates on the roid for killing monsters. Nobody is going to label you as a power gamer for it considering how no one labels the current ones who do that as power gamers. And you wouldn't be considered a power gamer because you have legitimate reason to have them at that time.
The pokemon thing is abusing laz injects and reviving a goliath over and over after you kill and butcher it's hides.
Here's a 400 IQ idea. Bring sci an extra tile worth of diamonds for a free, yes ABSOLUTELY FREE (NO CASH REQUIRED [THIS MEANS A TOTAL CREDIT VALUE OF ZERO]), laser cannon or two which you can open up on the roid with weak gibbtonite and you didn't have to do anything other than your normal job. If you're a miner with no science around what's the fucking point of you even existing since all the ore isn't going to anyone? Go play Motherload or Minecraft then you autist.
Maybe I'm retarded but how is getting a stupidly large amount of credits "fun" if they don't do anything even according to you? If getting big numbers was the main fun with miner then we would have gone with the miner points option rather than this credit rebalance but look at where we are now.

**Comment by FudgePucker** [comment]
@DrSnips Protolathe weapons are nerfed to shit if you open them by brute force (Unless that wasn't merged or was reverted)

If you think you are going to convince the captain warden or HoS to open a lock box so you can kill mobs you are mentally deficient or never play the game.

**Comment by DrSnips** [comment]
Are you living under a rock? That was reverted a long time ago. I think more than a year now.

**Comment by FudgePucker** [comment]
Ah, okay then.
I have been under a rock for about a year.
My b, I guess I am the one who is mentally deficient.

**Comment by SonixApache** [comment]
mapdiff stuck

**Comment by Kurfursten** [comment]
That's okay, I don't think this is going through as is. Contrary to what people (seem to) believe, I do want a solution that doesn't ruin miners. I liked the suggestions of PKAs in the protolathe and either a voucher system or biogenerator-esque smelter, but that all needs coding first. Personal life has kept me busy lately besides doing some light review and administrative stuff.

We've dealt with the miner distortion for years, a little longer won't be too bad.

**Comment by jknpj** [comment]
Whenst

**Comment by JohnExplode** [comment]
Neat

**Comment by jknpj** [comment]
Let's see how this goes instead of theorycrafting to death.

**Comment by Kurfursten** [comment]
reee now I have to hurriedly make the PKA accommodations I promised and I leave for dinner with the GF in 20 minutes

I will have a support PR for our miners up *tonight*

**Comment by agentofzog2** [comment]
So you make 1/10th of what you used to but most of the prices have been reduced by two thirds or so and you need multiple hauls to afford a PKA.
cool
epic


And the real issue is that the cart is being put in front of the horse in regards to having a working economy since this just  removes the only real way to make money and as far as I see there's nothing coming up to fix it besides the dumb black market thing.

**Comment by FudgePucker** [comment]
When's the PR to remove the slot machines?

**Pull request merged**